rebar.config

```diff
@@ -17,17 +17,9 @@
 
 {deps, [{katana_code, "1.1.2"}]}.
 
-{project_plugins,
- [{rebar3_hex, "~> 6.11.6"}, {rebar3_lint, "~> 0.4.1"}, {rebar3_hank, "~> 1.1.3"}]}.
-
-{dialyzer,
- [{warnings, [no_return, unmatched_returns, error_handling, underspecs]},
-  {get_warnings, true},
-  {plt_apps, top_level_deps},
-  {plt_extra_apps, []},
-  {plt_location, local},
-  {base_plt_apps, [erts, kernel, stdlib, runtime_tools, syntax_tools]},
```

Comment:
`runtime_tools, syntax_tools` moved to `.app.src`; the rest is not really required.